app/react/Viewer/components/Viewer.js

```diff
@@ -96,6 +96,8 @@ export class Viewer extends Component {
       className += ' connections';
     }
 
+    className += ' with-header';
```

Comment:
There is no need for this extra line, line 87 should read:
`let className = 'document-viewer with-header';`